src/test/java/org/folio/circulation/support/utils/DateFormatUtilTests.java

```diff
@@ -128,12 +134,10 @@ void shouldParseZonedDateTime() {
   @CsvSource(value = {
     "0, null, null, " + FORMATTED_DATE_TIME_NOW,
     "1, Z, null, " + FORMATTED_DATE_TIME_NOW,
-    "2, null, " + FORMATTED_DATE_TIME + ", " + FORMATTED_DATE_TIME,
-    "3, Z, " + FORMATTED_DATE_TIME + ", " + FORMATTED_DATE_TIME
+    "2, null, 2010-10-10T10:10:10.000Z, 2010-10-10T10:10:10.000Z",
+    "3, Z, 2010-10-10T10:10:10.000Z, 2010-10-10T10:10:10.000Z"
```

Comment:
Why do these lines no longer used the constants when the others do?

Reply:
I was simplifying the approach to make the tests more readable.

The use of the constant `FORMATTED_DATE_TIME`, made the code more consistent, but the variables are harder to read.
The values here are actually isolated and don't need to be shared.
Only the constants that using the system clock (ending in `_NOW`) are the ones that I think need to retain that behavior.

Reply:
Thanks for explaining this. I prefer inline constants so I'm happy about the change.